python/ql/src/experimental/Security/CWE-022/zipslip_bad.py

```diff
@@ -0,0 +1,9 @@
+import zipfile
+import shutil
+
+
+zf = zipfile.ZipFile(filename)
+with zf.open() as zipf:
+    #BAD : This could write any file on the filesystem.
+    for entry in zipf:
+    shutil.copyfileobj(entry, "/tmp/unpack/")
```

Comment:
cf. my earlier comment about `copyfileobj`, does this code actually do what you intend it to do?